sympy/physics/units/__init__.py

```diff
@@ -235,37 +235,48 @@ def find_unit(quantity, unit_system="SI"):
     >>> u.find_unit(u.charge)
     ['C', 'coulomb', 'coulombs', 'planck_charge', 'elementary_charge']
     >>> u.find_unit("ampere")
-    ['ampere', 'amperes']
-    >>> u.find_unit('angstrom')
-    ['angstrom', 'angstroms']
+    ['A', 'ampere', 'amperes', 'planck_current']
     >>> u.find_unit('volt')
-    ['volt', 'volts', 'electronvolt', 'electronvolts', 'planck_voltage']
+    ['V', 'v', 'volt', 'volts', 'planck_voltage']
```

Comment:
`electronvolt` should be in this list - it doesn't matter if the underlying unit has "volt" in it.